### DISCUSSION TITLE
Flux and Pagespeed Insights

### DISCUSSION THREAD

**Discussion by holubar** [livewire/flux#791] [Feature Requests]
I am working on website and when analyzing page speed, I get low score on mobile connected to flux scripts. url is benzing.sk, am I doing something wrong?

![image](https://github.com/user-attachments/assets/b3215523-08cd-424f-8e38-a9d45ade4aee)

**Comment by joshhanley** [comment]
@holubar this doesn't seem like a feature request to me. Can you please open an issue?
